source/manual/datalabs-bigquery.html.md

```diff
@@ -0,0 +1,86 @@
+---
+owner_slack: "#govuk-data-labs"
+title: Use Google BigQuery
+section: Data Labs Team Manual
+layout: manual_layout
+parent: "/manual.html"
+---
+
+
+Google BigQuery is a cloud-based SQL database. This database stores most GOV.UK visitor data. This data is primarily visitor journey data from visitors navigating through GOV.UK. BigQuery only stores visitor data if that visitor has accepted cookies.
+
+## How BigQuery data is populated
+
+Universal Analytics (UA) sends GOV.UK visitor journey data to the `govuk-bigquery-analytics.87773428.ga_sessions_intraday_YYYYMMDD` table 3 times a day, where:
+
+- `govuk-bigquery-analytics` is the project ID
+- `87773428` is the dataset name
+- `ga_sessions_intraday_YYYYMMDD` is the table name
+- `YYYYMMDD` is the current date in year-month-day format
+
+At the end of the day, UA automatically moves the data in this intraday table to a `ga_sessions_YYYYMMDD` table, and creates a new intraday table for the next day.
```

Comment:
does move imply the "cut and paste" rather than "copy and paste" of the table; might want to be more explicit that the intraday table will no longer be there (I'm not sure what the rules are on this)

perhaps this is too complex and the level of detail here is fine, as it's covered later

Reply:
will clarify